virttest/qemu_monitor.py

```diff
@@ -2038,3 +2038,32 @@ def qom_get(self, path, qproperty):
         self.verify_supported_cmd(cmd)
         args = {"path": path, "property": qproperty}
         return self.cmd(cmd, args)
+
+    def set_migrate_capability(self, state, capability):
+        """
+        Set the capability of migrate to state.
+
+        :param state: value of capability.
+        :param capability: capability which need to set.
+        """
+        cmd = "migrate-set-capabilities"
+        self.verify_supported_cmd(cmd)
+        args = {"capabilities":[{"state": state, "capability": capability}]}
+        return self.cmd(cmd, args)
+
+    def get_migrate_capability(self, capability=None):
+        """
+        Get the state of migrate-capability.
+
+        :param capability: capability which need to get.
+        :return: the state of migrate-capability, if capability=None,
+                 will return the list of query-migrate-capabilities.
+        """
```

Comment:
Hi,

Please optimize your logical, function name no match what you do in your function.
I guess it should be:
for info in self.query("migrate-capabilities"):
      if info["capability"] == capability:
           return info["state"]
 return False